core/src/components/item-sliding/item-sliding.tsx

```diff
@@ -252,7 +253,23 @@ export class ItemSliding implements ComponentInterface {
     return !!(this.rightOptions || this.leftOptions);
   }
 
+  private toggleContentScrollY() {
+    const content = this.el.closest('ion-content');
+    if (content === null) { return }
+
+    if (this.initialContentScrollY === undefined) {
+      this.initialContentScrollY = content.scrollY;
+      content.scrollY = false;
+    } else {
+      content.scrollY = this.initialContentScrollY;
+      this.initialContentScrollY = undefined;
+    }
+  }
+
   private onStart() {
+    // Prevent scrolling during gesture
+    this.toggleContentScrollY();
```

Comment:
Any chance we could make these disable/enable functions rather than a toggle? I think it's a bit easier to read that way.

Reply:
Made this change and I agree it is much more readable, ty.